hw-latest/fpga/fpga_configuration.tcl

```diff
@@ -0,0 +1,247 @@
+# Clean and create output directory.
+set outputDir ./caliptra_build
+set packageDir $outputDir/caliptra_tcl_package
+file delete -force $outputDir
+file mkdir $outputDir
+file mkdir $packageDir
+
+# Path to rtl
+set rtlDir ../caliptra-rtl-fpga
+
+# JTAG enabled
+set JTAG 1
+
+# Set Verilog defines to:
+#     Make Caliptra use an icg that doesn't clock gate
+#     Make the VEER core be optimized for FPGA (no clock gating)
+#     Define VEER TEC_RV_ICG to allow beh_lib to synthesise without error
+set VERILOG_OPTIONS {TECH_SPECIFIC_ICG USER_ICG=fpga_fake_icg RV_FPGA_OPTIMIZE TEC_RV_ICG=clockhdr}
+if {FALSE} {
+  # Add option to use Caliptra's internal TRNG instead of ETRNG
+  lappend VERILOG_OPTIONS CALIPTRA_INTERNAL_TRNG
+}
+
+# Start the Vivado GUI for interactive debug
+start_gui
```

Comment:
Is this part necessary? I'd really like to be able to run this as part of CI, which doesn't have an x server.

Reply:
No, my tentative plan is to have a common tcl file doing all the setup, then separate GUI and automation tcl files.

Reply:
Making this configurable.